README.md

```diff
@@ -89,7 +89,7 @@ tar zxvf millicast-cli-1.0.0-Darwin.tar.gz
 
 cd millicast-cli-1.0.0-Darwin
 
-chmod +x install.sh
+sudo chmod +x install.sh
```

Comment:
Why sudo ?
You do not need sudo here

Reply:
You're right. The `sudo` is required to execute the installation file (the next line). I'll make another commmit to reflect this change.